dace/frontend/fortran/fortran_parser.py

```diff
@@ -3263,8 +3263,17 @@ def name_and_rename_dict_creator(parse_order: list,dep_graph:nx.DiGraph)->Tuple[
     return name_dict, rename_dict
 
 
-def create_sdfg_from_fortran_file_with_options(source_string: str, source_list, include_list, icon_sources_dir,
-                                               icon_sdfgs_dir, normalize_offsets: bool = False, propagation_info=None):
+def create_sdfg_from_fortran_file_with_options(
+    source_string: str,
+    source_list, include_list,
+    sdfgs_dir,
+    subroutine_name: Optional[str] = None,
+    normalize_offsets: bool = True, 
+    propagation_info=None,
+    enum_propagator_files: List[str] = None,
```

Comment:
`List[str] = None` probably should be `Optional[List[str]] = None` since `None` cannot be a default value for list.